clusterloader2/pkg/tuningset/global_qps_load.go

```diff
@@ -59,7 +60,10 @@ func newGlobalQPSLoad(params *api.GlobalQPSLoad) *globalQPSLoad {
 func (ql *globalQPSLoad) Execute(actions []func()) {
 	var wg wait.Group
 	for i := range actions {
-		ql.limiter.Wait(context.TODO())
+		err := ql.limiter.Wait(context.TODO())
+		if err != nil {
+			log.Fatal("QPS load exec error ", err)
```

Comment:
we don't use log in clusterloader - let's use klog if you need

that said - I don't think it deserves fatal - let's simply log an error.

Reply:
Added klog.Errorf